quantize.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/python3
+
+"""Script to execute quantization on a given model."""
+
+import subprocess
+import argparse
+import sys
+import os
+
+
+def main():
+    """Parse the command line arguments and execute the script."""
+
+    parser = argparse.ArgumentParser(
+        prog='Quantization Script',
+        description='This script quantizes a model.'
+    )
+
+    parser.add_argument("models", nargs='+', dest='models')
+    parser.add_argument(
+        '-r', '--remove-16', action='store_true', dest='remove_f16',
+        help='Remove the f16 model after quantizing it.'
+    )
+
+    args = parser.parse_args()
+
+    for model in args.models:
+
+        model_path = os.path.join("models", model, "ggml-model-f16.bin")
+
+        for i in os.listdir(model_path):
```

Comment:
there's another PR about parallelizing the quantizations. here it would be easy just to wrap this in a `multiprocessing.Pool()` (with the subprocess calls extracted out to a top-level function): https://docs.python.org/3/library/multiprocessing.html

Reply:
I don't understand exactly what can be parallelized here, is it quantizing many models at the same time? The code from the latest commit removed the loop that acts upon the incorrect listing of the file (`for i in os.listdir(model_path)`) by the way.